server/vulnerabilities/msrc/analyzer_test.go

```diff
@@ -67,37 +50,7 @@ func TestAnalyzer(t *testing.T) {
 			vfA.ProductIDs["123"] = true
 			b.VendorFixes[456] = vfA
 
-			updates := []fleet.WindowsUpdate{
-				{KBID: 789},
-			}
-
-			require.True(t, patched(op, b, b.Vulnerabities["cve-123"], pIDs, updates))
-		})
-
-		t.Run("remediated by a cumulative update", func(t *testing.T) {
-			b := parsed.NewSecurityBulletin(prod.Name())
-			b.Products["123"] = prod
-			pIDs := map[string]bool{"123": true}
-
-			vuln := parsed.NewVulnerability(nil)
-			vuln.RemediatedBy[456] = true
-			b.Vulnerabities["cve-123"] = vuln
-
-			vfA := parsed.NewVendorFix("10.0.22000.796")
-			vfA.Supersedes = ptr.Uint(123)
-			vfA.ProductIDs["123"] = true
-			b.VendorFixes[456] = vfA
-
-			vfB := parsed.NewVendorFix("10.0.22000.796")
-			vfB.Supersedes = ptr.Uint(456)
-			vfB.ProductIDs["123"] = true
-			b.VendorFixes[789] = vfA
-
-			updates := []fleet.WindowsUpdate{
-				{KBID: 789},
-			}
-
-			require.True(t, patched(op, b, b.Vulnerabities["cve-123"], pIDs, updates))
+			require.True(t, patched(op, b, b.Vulnerabities["cve-123"], pIDs))
```

Comment:
Is it interesting to test this with multiple vulnerabilities, not just one?

Reply:
Not necessarily in this case.  `patched()` expects a single `msrc.Vulnerability` type